src/components/AddressSearch/index.js

```diff
@@ -1,4 +1,4 @@
-import lodashGet from 'lodash/get';
+import {filter, get as lodashGet, some} from 'lodash';
```

Comment:
```suggestion
import _ from 'lodash';
```

Just so the usage of `lodash` is consistent with the rest of the codebase